cli/azd/cmd/show.go

```diff
@@ -110,8 +110,8 @@ func (s *showAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 	resourceManager := infra.NewAzureResourceManager(ctx)
 
 	if resourceGroupName, err := resourceManager.FindResourceGroupForEnvironment(ctx, env); err == nil {
-		for name := range prj.Services {
-			if resources, err := project.GetServiceResources(ctx, resourceGroupName, name, env); err == nil {
+		for name, serviceConfig := range prj.Services {
+			if resources, err := serviceConfig.GetServiceResources(ctx, resourceGroupName, env); err == nil {
```

Comment:
@ellismg When I was fixing the service resource validation, I found that `azd show` would use the static method `GetServiceResources` which would always search by the `azd-service-name` tag, as opposed to first looking at the `resourceName` element in `azure.yaml`. This change restores consistency but let me know if you think this should work differently.

Reply:
My bad - this seems right!